app/src/renderer/views/Inbox/MainContent/Conversation/Item/File.tsx

```diff
@@ -181,9 +249,11 @@ function CompleteFile(item: Item) {
   const formattedFilename = formatFilename(filename, filenameMaxLength, 6);
   const tooltipTitle = filename.length > filenameMaxLength ? filename : "";
 
+  const { Icon, color } = getFileIconAndColor(filename);
+
   return (
     <div className="flex items-center justify-start mt-2 mb-2">
-      <FileCheck2 size={30} strokeWidth={1} className="file-icon" />
+      <Icon style={{ fontSize: 30, color }} className="file-icon" />
```

Comment:
looking at the ant docs, it looks like the audio/video from the Figma docs comes from this `Attachment` component? rather than just using the icons: https://x.ant.design/components/attachments#attachments-demo-files 

a little strange that the icons don't 1:1 map, but may be easier to just use the `Attachment` component, since it looks like then we get the extension <> icon mapping for free